lib/adm.js

```diff
@@ -5469,8 +5470,24 @@ maAdm.prototype.doExecPlan = function(sout, serr, dryrun, callback) {
                 }
 
                 inputs = Object.keys(self.ma_plan[svcname]);
-                concurrency = dryrun ? 1 : inputs.length;
```

Comment:
This section is the fix. Other bits are for "Also:..." unrelated stuff while I was here.